data.json

```diff
@@ -1,5 +1,9 @@
 {
     "restaurants": [
+        {
+            "name": "Rosemary Rocksalt",
+            "weight": 9001
```

Comment:
This seems a bit high...

Reply:
Agreed. Rosemary Rocksalt is OP. Please nerf.

Reply:
Would 4 be a more reasonable value?

Reply:
I think 4 would do fine

Reply:
Missed opportunity for a "it's over 9000!" reference. smh.